packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -410,7 +410,14 @@ function hookFunctionsSave(parentProcessMode?: string): IWorkflowExecuteHooks {
 					workflowId: this.workflowData.id,
 				});
 
-				await restoreBinaryDataId(fullRunData, this.executionId, this.mode);
+				try {
+					await restoreBinaryDataId(fullRunData, this.executionId, this.mode);
+				} catch (e) {
+					logger.error('Failed to restore binary data ID', {
+						executionId: this.executionId,
+						mode: this.mode,
+					});
```

Comment:
We are still ignoring the thrown error. Could we at least include it in the logging so we know what the underlying reason for the failure was?

Reply:
@ivov added your suggestions and I rebased my branch to contain it